Doc/library/ast.rst

```diff
@@ -1748,6 +1748,34 @@ and classes for traversing abstract syntax trees:
       Added the *indent* option.
 
 
+.. _ast-compiler-flags:
+
+Compiler Flags
```

Comment:
I understand why you documented these here, but I am worried about the discovery of these flags. Should it be a cross-reference in the `compile` function docs? Are these even mentioned there?

Reply:
IMHO this is the initial step to increase that `non-existent` discoverability. I believe we may need to re-write whole flags (and dont_inherit) documentation again for describing them supporting not only future features, but compiler flags in an another PR.

Reply:
Giving that this PR is rather small, I would suggest doing this in this PR as the decisions we take there may affect this work (for instance we may decide that these flags will be documented only on the `compile` function, rendering this work obsolete.

Reply:
I made the required adjustments.